Documentation/ceph-cluster-crd.md

```diff
@@ -1390,6 +1391,21 @@ spec:
     #externalMgrEndpoints:
       #- ip: 192.168.39.182
     #externalMgrPrometheusPort: 9283
+    # prometheus rule alerts values for overriding default prometheus rules values
+    # optional: specific alerts can be disabled by setting disabled field to true 
+    #alertRuleOverrides:
+    #  CephNodeDown:
+    #    disabled: true
+    #  CephMgrIsAbsent:
+    #    for: 1m
+    #    severityLevel: <custom-severityLevel> #<custom-severityLevel> must be one of the next levels: warning, critical, error
```

Comment:
how about we just use one of the examples values here?
```suggestion
    #    severityLevel: warning # must be warning, critical, or error
```

Reply:
Sure, make sense.

Reply:
Regarding the suggestion about enum on both of the fields,  `severity` and `severityLevel` are not exactly the same. `severity` is a label, so we can use a custom severity label in Prometheus. It might help if you would like to direct different rook alerts to different slack channels.

For example,
You can give some of the alerts the  `severity: rook-warning`, so all the alerts with this label can be sent to a different channel that is separated from alerts with the  `severity: warning`.

I believe that in `severity` field, we should let the user decide which severity label to use.